build.gradle

```diff
@@ -64,11 +65,13 @@ configure(allProjs) {
         scalaGraphVersion = '1.12.5'
         scalafmtVersion = '1.5.1'
         hadoopVersion = 'hadoop2'
+        scalajCollVersion = '0.1.2'
```

Comment:
Where is this being used?

Reply:
This shouldn't be there (It was from a version that was 1~2 days earlier than when I forked the repo). I'll fix it in the new PR.